### ISSUE TITLE
Basic petstore example is not working (version 0.26.0)

### ISSUE THREAD

**Issue by pontusvision** [pmcelhaney/counterfact#580]
The following example isn't working:
```
npx counterfact@latest https://petstore3.swagger.io/api/v3/openapi.yaml api --open
Need to install the following packages:
counterfact@0.26.0
Ok to proceed? (y) y
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated json-schema-ref-parser@9.0.9: Please switch to @apidevtools/json-schema-ref-parser
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated json-schema-ref-parser@6.1.0: Please switch to @apidevtools/json-schema-ref-parser
$ echo $?
127
node --version
v16.19.0

```
The error code 127 (-1) indicates a failure in the npx script, and no files are generated.

**Comment by dethell-jh** [comment]
Looks like version `0.26.0` is having install issues. Gettin a `patch-package` error when trying to update it as an installed dependency in my own project and using `npx counterfact@latest` just never starts.

**Comment by pmcelhaney** [comment]
Yikes! For now change `counterfact@latest` to `counterfact@0.25`.

**Comment by pmcelhaney** [comment]
Oh, it's not even installing. There's a postinstall script to run patch-package and for some reason the patch-package command is not found. 

patch-package is only needed at build time so that the latest versions of ESLint and all the plugins can run. I removed the postinstall entry. That should fix it. 

#583

**Comment by pmcelhaney** [comment]
Fixed in 0.26.1. Thanks for letting me know!
